ocs_ci/deployment/vmware.py

```diff
@@ -733,6 +734,15 @@ def destroy_cluster(self, log_level="DEBUG"):
         logger.debug(f"changing state from {str_to_modify} to {target_str}")
         replace_content_in_file(terraform_tfstate, str_to_modify, target_str)
 
+        # remove csi users in case of external deployment
+        if config.DEPLOYMENT["external_mode"]:
```

Comment:
Do we need to do that also for vsphere-cleanup script?

Reply:
yes, we need to remove in cleanup script as well in case of external mode. I will raise github issue for the same